tests/ImageSharp.Tests/Formats/Tiff/TiffDecoderTests.cs

```diff
@@ -173,6 +203,32 @@ public void TiffDecoder_CanDecode_42Bit<TPixel>(TestImageProvider<TPixel> provid
         public void TiffDecoder_CanDecode_48Bit<TPixel>(TestImageProvider<TPixel> provider)
             where TPixel : unmanaged, IPixel<TPixel> => TestTiffDecoder(provider);
 
+        [Theory]
+        [WithFile(FlowerRgb242424Contiguous, PixelTypes.Rgba32)]
+        [WithFile(FlowerRgb242424ContiguousLittleEndian, PixelTypes.Rgba32)]
+        [WithFile(FlowerRgb242424Planar, PixelTypes.Rgba32)]
+        [WithFile(FlowerRgb242424PlanarLittleEndian, PixelTypes.Rgba32)]
+        public void TiffDecoder_CanDecode_72Bit<TPixel>(TestImageProvider<TPixel> provider)
+            where TPixel : unmanaged, IPixel<TPixel>
+        {
+            // Note: because the MagickReferenceDecoder fails to load the image, we only debug save them.
+            using Image<TPixel> image = provider.GetImage();
+            image.DebugSave(provider);
+        }
+
+        [Theory]
+        [WithFile(FlowerRgb323232Contiguous, PixelTypes.Rgba32)]
+        [WithFile(FlowerRgb323232ContiguousLittleEndian, PixelTypes.Rgba32)]
+        [WithFile(FlowerRgb323232Planar, PixelTypes.Rgba32)]
+        [WithFile(FlowerRgb323232PlanarLittleEndian, PixelTypes.Rgba32)]
+        public void TiffDecoder_CanDecode_96Bit<TPixel>(TestImageProvider<TPixel> provider)
+            where TPixel : unmanaged, IPixel<TPixel>
+        {
+            // Note: because the MagickReferenceDecoder fails to load the image, we only debug save them.
```

Comment:
Magick.Net can actually decode the images, but in `MagickReferenceDecoder` I am not sure how to correctly transfer the pixel byte data from Magick.Net to our format.

Reply:
@dlemstra Would you be able to assist here?

Reply:
I will go ahead and merge this now. It would be still useful to know if we can use Magick.Net for images with color channel bit depth >= 24. We can add this in a future PR, if  it is possible.